src/popover/popover.ts

```diff
@@ -251,6 +244,12 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
       // mark the parent component to be checked.
       this._windowRef.changeDetectorRef.markForCheck();
 
+      this._ngZone.runOutsideAngular(() => {
```

Comment:
I think we should do (as a generic algoritm for all components, not only popover):
- create a new popper on opening. outside angular
- when zone is stable AND popup is opened do `popper.update()`
- when popup is closed release popper resources

We should probably rename `positionELements` to someting like `createPopper` and call it only initially and then use the returned popper instance inside the component to call `.update()` Just an idea.